aries-site/src/pages/templates/drill-down-navigation.mdx

```diff
@@ -1,3 +1,50 @@
-import { ComingSoon } from '../../components';
+import { DiagramPreview, Example } from '../../layouts';
+import { DrillDownCardExample, DrillDownDataTableExample, DrillDownListExample } from '../../examples';
 
-<ComingSoon />
\ No newline at end of file
+## When to Use:
+- When users need to focus on one task at a time.
+- To give information overview with the ability to drill down into specific information.
+- When screen real estate is limited.
+- When designing for many levels of functionality.
+- To divide meaningful content into sections.
+
+## Guidelines:
+- When the user is moving forward step-by-step to access further information.
+- When going from your Account > Dashboard > Service.
```

Comment:
Folks tend to misuse "dashboard," trying to avoid reinforcing the misuse of that language, but perhaps that ship has sailed and is a lost cause... Tagging @bahriaditi for her thoughts.

```suggestion
- When going from your Account > Services > Service.
```

```suggestion
- When going from your Account > Dashboard > Report Detail.
```

Reply:
I agree with @halocline, we are trying to steer clear of having dashboards being used as a homepage/ gateway. The suggested changes make sense to me ✔️